src/test/java/de/tum/in/www1/artemis/assessment/GradingScaleUtilService.java

```diff
@@ -27,6 +27,13 @@ public class GradingScaleUtilService {
     @Autowired
     private GradingScaleRepository gradingScaleRepository;
 
+    /**
+     * Creates a set of grade steps.
```

Comment:
Here you could also explain how many grade steps are created and that one is a failing one and the other a passing one.